test/sequential/test-fs-opendir-recursive.js

```diff
@@ -128,7 +128,7 @@ for (let i = 0; i < expected.length; i++) {
 }
 
 function getDirentPath(dirent) {
-  return pathModule.relative(testDir, dirent.path);
+  return pathModule.relative(testDir, pathModule.join(dirent.path, dirent.name));
```

Comment:
This change is not OK, we cannot make this kind of breaking change, especially on a maintenance LTS line. This needs to remain as it was on v18.19.1:

https://github.com/nodejs/node/blob/6610cc42ff50209248dd46d1ef947d1702fdf108/test/sequential/test-fs-opendir-recursive.js#L131

Reply:
I got it!  I fixed in [b2d61d8](https://github.com/nodejs/node/pull/53969/commits/b2d61d8411613a33bfd35fd284e1ed7d8d837334).
Many Thanks!